src/api/client/resolvers.ts

```diff
@@ -143,7 +143,7 @@ export const queryNodeStitchingResolvers = (
           views: viewsLookup[channel.id]?.views || 0,
         }))
       } catch (error) {
```

Comment:
Let's use a similar strategy as with the previous resolver:
- channels fetch outside of try
- separate independent try-catch for views and follows with separate messages
- `channels resolver` as source